spec/requests/lobbies_spec.rb

```diff
@@ -226,4 +226,67 @@
       expect(Pusher).to have_received(:trigger).with(lobby.code, Lobby::PLAYER_JOIN, { id: Player.last.id })
     end
   end
+
+  describe 'start' do
+    let(:quiz) { FactoryBot.create(:quiz, user: user) }
+    let(:lobby) { FactoryBot.create(:lobby, quiz: quiz) }
+
+    before do
+      allow(Pusher).to receive(:trigger)
+    end
+
+    subject { post start_api_v1_lobby_path(id: lobby.id) }
+    
+    it 'responds with successful HTTP status' do
+      subject
+
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'updated lobby status to in_progress' do
+      subject
+
+      expect(lobby.reload.status).to eq('in_progress')
+    end
+
+    it 'triggers a LOBBY_START Pusher event' do
+      subject
+
+      expect(Pusher).to have_received(:trigger).with(lobby.code, Lobby::LOBBY_START, {})
+    end
+
+  end
+
+  describe 'answer' do
```

Comment:
You could also test the creation of `PlayerAnswer` models. And the fact that if some already existed, they're destroyed.